.gitignore

```diff
@@ -82,6 +82,7 @@ docs/*/pdfreactor-fonts.html
 #################
 *.egg-info
 .installed.cfg
+static/
```

Comment:
That's right but when django runs `collectstatic` it will grab all the assets from all the apps (we really only have 1 we care about now but that should change as time goes on) and shove them into the /static/ folder. When django looks for static assets it will look there (depending on if DEBUG is turned off or not. Ross mentioned a couple days ago that DEBUG turned on fakes this behavior so that static assets in the app can live in the app but still be rendered). 

Sorry for the parens. :cactus: